test/presubmit-tests.sh

```diff
@@ -27,10 +27,45 @@ export DISABLE_MD_LINTING=1
 
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
 
-function post_build_tests() {
-    golangci-lint run
+
+unit_tests() {
+ :
+}
+
+
+build() {
+  GO111MODULE=on operator-sdk build gcr.io/tekton-nightly/tektoncd-operator
+}
+
+install_operator_sdk() {
+  local sdk_rel="v0.9.0"
+  mkdir -p tmp/bin/
+
+  curl -JL \
+    https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu \
+    -o tmp/bin/operator-sdk
+  chmod +x tmp/bin/operator-sdk
+}
+
+post_build_tests() {
+  ## TODO: enable this back on the next pr
+  #golangci-lint run
+  GO111MODULE=on operator-sdk test local ./test/e2e  \
```

Comment:
This should be in an integration-tests function instead of a post build. The build job only build, the e2e tests are meant to run in the intergration job (in parallel with build) 👼